submissions/OlgaBarnikova/Popup/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Popup</title>
+        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0">
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">  
+                <div class="block_menu">
+                    <nav class="navigation">
+                        <a href="#" class="popup_icons">Proekt</a>
+                        <a href="#" class="popup_icons">About us</a>
+                        <label for="popup">
+                            <img src="popup-icons/popup-button.png" alt="" class="popup_block_img" />                            
+                        </label>
```

Comment:
Users must have the ability to open a popup with a keyboard. [Watch video](https://share.vidyard.com/watch/ZDCPyrd5EAnKvP895smxQc?)